README.md

```diff
@@ -209,6 +209,13 @@ putting it in `/opt/docker/zulip/zulip/certs/` (by default, the
 `zulip` container startup script will generate a self-signed certificate and
 install it in that directory).
 
+**Load balancer**. To tell Zulip it's behind a loadbalancer, you can set
```

Comment:
There are still two more instances of “loadbalancer” to be fixed to “load balancer”.